docs/standard/containerized-lifecycle-architecture/Microsoft-platform-tools-containerized-apps/index.md

```diff
@@ -28,30 +29,30 @@ Table 3-1: Open DevOps workflow to any technology
+- **Platform for Docker Apps development** The development of a service, or collection of services that make up an "app." The development platform provides all the work developers requires prior to pushing their code to a shared code repository. Developing services, deployed as containers, are similar to the development of the same apps or services without Docker. You continue to use your preferred language (.NET, Node.js, Go, etc.) and preferred editor or IDE like Visual Studio or Visual Studio Code. However, rather than consider Docker a deployment destination, you develop your services in the Docker environment. You build, run, test, and debug your code in containers locally, providing the destination environment at development time. By providing the destination environment locally, Docker containers set up what will drastically help you improve your DevOps life cycle. Visual Studio and Visual Studio Code have extensions to integrate Docker containers within your development process.
 
--   **DevOps for Docker Apps** Developers creating Docker applications can use Azure DevOps Services or any other third-party product, like Jenkins, to build out a comprehensive automated application life cycle management (ALM).
+- **DevOps for Docker Apps** Developers creating Docker applications can use [Azure DevOps Services](https://azure.microsoft.com/services/devops/) or any other third-party product, like Jenkins, to build out a comprehensive automated application life-cycle management (ALM).
 
-With Azure DevOps Services, developers can create container-focused DevOps for a fast, iterative process that covers source-code control from anywhere (Azure DevOps Services-Git, GitHub, any remote Git repository, or Subversion), Continuous Integration (CI), internal unit tests, inter container/service integration tests, Continuous Delivery (CD), and release management (RM). Developers also can automate their Docker application releases into Azure Container Service, from development to staging and production environments.
- 
--   IT production management and monitoring.
+  With Azure DevOps Services, developers can create container-focused DevOps for a fast, iterative process that covers source-code control from anywhere (Azure DevOps Services-Git, GitHub, any remote Git repository, or Subversion), Continuous Integration (CI), internal unit tests, inter container/service integration tests, Continuous Delivery (CD), and release management (RM). Developers also can automate their Docker application releases into Azure Container Service, from development to staging and production environments.
 
-**Management** IT can manage production applications and services in several ways:
+- IT production management and monitoring.
```

Comment:
This should follow the same format as the previous bulleted list items. "IT production management and monitoring." should be bolded, there should be no paragraph break between the bolded text and "Management IT can manage.., and "Management" in "Management IT..." should not be bolded.

Reply:
Ok, I'll rearrange this after committing the suggestions.